pilot/lib/pilot/build_manager.rb

```diff
@@ -366,6 +366,9 @@ def expire_previous_builds(build)
     # If there are multiple teams, infer the provider from the selected team name.
     # If there are fewer than two teams, don't infer the provider.
     def transporter_for_selected_team(options)
+      # Ensure that `Spaceship::ConnectAPI.token` is set correctly, if required
+      login
```

Comment:
I was looking at how `distribute` and `list` do this and they actually call `start(options)` so maybe we do that as well here 🤷‍♂️ 

```suggestion
      start(options)
```